app/main.py

```diff
@@ -42,5 +56,8 @@ def wash_single_car(self, car: Car) -> None:
     def rate_service(self, rating: int | float) -> None:
         total_rating = self.average_rating * self.count_of_ratings
         self.count_of_ratings += 1
-        self.average_rating = round((total_rating + rating)
-                                    / self.count_of_ratings, 1)
+        self.average_rating = (round(
```

Comment:
round should be on the next line
the general rule is: content inside brackets should be on other lines than these brackets